libhybris/hybris/egl/platforms/hwcomposer/hwcomposer.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2015 Jolla Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef HYBRIS_HWCOMPOSER_H
+#define HYBRIS_HWCOMPOSER_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+struct ANativeWindow;
+struct ANativeWindowBuffer;
+
+typedef void (*present_callback)(void *user_data, struct ANativeWindow *window, struct ANativeWindowBuffer *buffer);
+
+struct ANativeWindow *createHWComposerNativeWindow(unsigned int width, unsigned int height, unsigned int format, present_callback present, void *cb_data);
+int HWCNativeBufferGetFence(struct ANativeWindowBuffer *buf);
```

Comment:
We're mixing naming conventions here. Either we put the verb at the beginning or the type of object we're dealing with.

HWComposerNativeWindowCreate
HWCNativeBufferGetFence
HWCNativeBufferSetFence

or

createHWComposerNativeWindow
getHWCNativeBufferFence
setHWCNativeBufferFence